README.md

```diff
@@ -94,6 +94,23 @@ emitter.send(new CloudEvent({ specversion: Version.V03, source, type }))
   }).catch(console.error);
 ```
 
+#### Cloud Events
```

Comment:
nit: CloudEvents does not have a space.

Maybe rename this section to `#### CloudEvent Objects`?